.spi.yml

```diff
@@ -3,15 +3,10 @@ builder:
   configs:
   - platform: ios
     scheme: ComposableArchitecture
-    swift_version: 5.9
```

Comment:
What `swift_version` does here is contains the `scheme` config to `ios` and `5.9` where previously it applied to _all_ iOS builds. You almost certainly don't want this here or in the other configs below.